localization/english/decisions/wc_get_loan_I_english.yml

```diff
@@ -0,0 +1,6 @@
+l_english:
```

Comment:
As I mentioned, make the descriptions similar to those of the Holy Order description with tweaks to mention Goblins. @Warcraft-GoA-Development-Team/writer / @Warcraft-GoA-Development-Team/english-proofreaders  can help with this if needed.